build-scripts/build-emscripten.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -exo pipefail
+
+emsdk install 3.1.51
+emsdk activate 3.1.51
+
+make -j`nproc` NATIVE=emscripten BACKTRACE=0 TILES=1 TESTS=0 RUNTESTS=0 RELEASE=1 cataclysm-tiles.js
```

Comment:
Curious if ccache works with emcc so we can reduce build times.

Reply:
Yeah this is definitely worth looking into.